src/main/java/jsr223/nativeshell/NativeShellRunner.java

```diff
@@ -122,6 +130,9 @@ private static int run(ProcessBuilder processBuilder, Reader processInput, Write
                 @Override
                 public void run() {
```

Comment:
i believe you can replace it with lambda

Reply:
source code compatibility is set to java 7

Reply:
as well spotless is not enabled, I guess we'll have to add this soon, but not today